Util/LegacyFormHelper.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the NelmioApiDocBundle.
+ *
+ * (c) Nelmio <hello@nelm.io>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Nelmio\ApiDocBundle\Util;
+
+/**
+ * Extracted from FOSUserBundle.
+ *
+ * @internal
+ */
+final class LegacyFormHelper
```

Comment:
Why you are not using short syntax here, it is not taken from original repository ;)